### PR TITLE
Implements #1

### PR DESCRIPTION
**Pull request by tma02**
Please test options with updateOption(key, value) in the background page console.

> updateOption(key, value)

< true: update in memory and local storage set() requested but not necessarily complete.

< false: option is disabled and was not updated.

### CODE REVIEW

**Comment by tma02** [comment]
There's no UI for options yet, we should be implementing that as part of #7, let me know if you need help updating options for testing.

**Comment by tma02** [comment]
Also I just realized there's no UI for setting a master password. We should still merge this so that one of you two (or someone else) can start working on the UI.

File: src/chromium/backgroundOptions.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+const defaultOptions = {
+  encryptKey: {
+    title: 'Store keys encrypted using master password',
+    info: 'Enabling this option will encrypt your secret key using your master password. This is neccessary for enabling Chrome sync storage.',
+    type: 'boolean',
+    default: false,
+    value: false,
+    disabled: (options) => {
+      return options.masterPassword.value.length == 0;
```
Comment (cata85) [inline src/chromium/backgroundOptions.js]: `Error in event handler for runtime.onMessage: TypeError: Cannot read property 'value' of undefined` Got this when I generated a new key.

Comment (tma02) [reply src/chromium/backgroundOptions.js]: Could I get steps to reproduce? Was unable to reproduce by

1. Loading extension on fresh Chrome
2. Navigating to https://twitter.com
3. Opening dev tools
4. Generating new secret keys

Icon update functionality remained as well, which leads me to believe this is the root of the #11

Comment (tma02) [reply src/chromium/backgroundOptions.js]: Did you mess with updateOptions() or the options in any way during testing?

File: src/chromium/twitter.js [hunk 2]
```diff
@@ -12,16 +12,12 @@ var latestReceivedMessage = {
   text: ''
 };
 
-chrome.storage.local.get(['publicKey', 'secretKey'], (result) => {
-  keyPair.publicKey = result.publicKey;
-  keyPair.secretKey = result.secretKey;
-});
+chrome.runtime.sendMessage({ action: 'getKeyPair' });
```
Comment (dominictassio) [inline src/chromium/twitter.js]: Move the sendMessage call to after the addListener call on line 17.

File: src/chromium/twitter.js [hunk 3]
```diff
@@ -38,19 +34,44 @@ function getMySecretKey() {
 }
 
 function encryptMessage(message) {
-  return HermesCrypto.encryptMessage(message, getTheirPublicKey(), getMySecretKey());
+  var encryptedMessage = '';
+  try {
+    encryptedMessage = HermesCrypto.encryptMessage(message, getTheirPublicKey(), getMySecretKey());
+  }
+  catch(e) {
+
```
Comment (dominictassio) [inline src/chromium/twitter.js]:
```suggestion
    return false;
```

File: src/chromium/backgroundOptions.js (hunk 1 above)
Comment (dominictassio) [reply src/chromium/backgroundOptions.js]: I could not reproduce this error. We should address this at a later time if it ever shown to be reproducible.

**Pull request merged**